packages/react-storage/src/components/StorageBrowser/views/LocationsView/useLocationsView.ts

```diff
@@ -0,0 +1,130 @@
+import { LocationAccess } from '../../context/types';
+import { ActionState } from '../../context/actions/createActionStateContext';
+import { useLocationsData } from '../../context/actions';
+import { useControl } from '../../context/control';
+import React from 'react';
+import { usePaginate } from '../hooks/usePaginate';
+
+interface UseLocationsView {
+  // items to display
+  items: LocationAccess[];
```

Comment:
@calebpollman this differs from the doc, but it was easier to consume processed items directly instead of calling a function.